src/libs/ReportUtils.ts

```diff
@@ -5852,18 +5852,17 @@ function getDeletedTransactionMessage(action: ReportAction) {
     return message;
 }
 
-function getReportDetails(reportID: string): {reportName: string; reportUrl: string} {
-    const report = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`];
+function getReportDetails(reportID: string, report?: Report): {reportName: string; reportUrl: string} {
```

Comment:
Why are we passing both reportID and report as params? We can access `reportID` via `report`